client/packages/invoices/src/OutboundShipment/DetailView/AddFromScannerButton.tsx

```diff
@@ -67,6 +68,20 @@ export const AddFromScannerButtonComponent = ({
     };
   }, []);
 
+  const label = t(isScanning ? 'button.stop' : 'button.scan');
+  useRegisterActions(
+    [
+      {
+        id: 'action:scan-barcode',
+        name: `${label} (ctrl+s)`,
```

Comment:
```suggestion
        name: `${label} (Ctrl+s)`,
```
More standard?

Reply:
Yeah, just looks uglier as all other shortcuts are lowercase 😉 
I don't care either way, so have changed. More importantly.. the shortcut shouldn't show when the button isn't available. Changed that too in ab47372dd